src/Parser/ArgumentsParser/ArgumentsParser.hpp

```diff
@@ -0,0 +1,55 @@
+/*
+** EPITECH PROJECT, 2022
+** PLAZAAAAAAAAAAAAA
+** File description:
+** ArgumentsParser
+*/
+
+#ifndef ARGUMENTSPARSER_HPP_
+#define ARGUMENTSPARSER_HPP_
+
+#include "../AParser.hpp"
```

Comment:
Relative include path will be removed in favor of a simpler `#include "AParser.hpp"`